src/client/componentsMealsharing/AllReservations.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import "../App.css";
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+function AllReservations() {
+  const [fetchData, setFetchData] = useState([]);
```

Comment:
Would be good to use a more descriptive name for the state for example: 
```suggestion
  const [reservations, setReservations] = useState([]);
```